src/ERC20DelegatesUpgradeable.sol

```diff
@@ -12,13 +12,13 @@ import {EIP712Upgradeable} from
 import {Initializable} from "lib/openzeppelin-contracts-upgradeable/contracts/proxy/utils/Initializable.sol";
 
 /// @title ERC20DelegatesUpgradeable
-/// @author Morpho Labs
+/// @author Morpho Association
```

Comment:
Why this change ? The author is Morpho Labs, but it is written for the association, right ?

Reply:
Yes + we want to transition more and more to the Association in general